tests/e2e/lib.sh

```diff
@@ -202,6 +202,20 @@ export_test_environment() {
         # GKE uses this network for services. Consider it as a private subnet.
         ci_export ROX_NON_AGGREGATED_NETWORKS "${ROX_NON_AGGREGATED_NETWORKS:-34.118.224.0/20}"
     fi
+
+    info "CI_JOB_NAME: ${CI_JOB_NAME}"
+    source "${SHARED_DIR}/dotenv"
+
+    ci_export CLUSTER_API_ENDPOINT "$CLUSTER_API_ENDPOINT"
+    ci_export CLUSTER_USERNAME "$CLUSTER_USERNAME"
+    ci_export CLUSTER_PASSWORD "$CLUSTER_PASSWORD"
+    
+    if [[ "${CI_JOB_NAME:-}" =~ ocp-4-[0-9]+-ui-e2e-tests ]]; then
```

Comment:
```suggestion
    if [[ "${CI_JOB_NAME:-}" =~ ocp-ui-e2e-tests ]]; then
```
We may not need this here, but if so